src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -23,7 +23,6 @@
 class SuluMediaExtension extends Extension
 {
     const DEFAULT_FORMAT_NAME = '170x170';
-    const DEFAULT_GHOST_SCRIPT_PATH = 'ghostscript';
```

Comment:
wanted to delete the constants to avoid that somebody use the constant instead of the parameter.